### PR TITLE
119 ios harcoded secrets

### PR DESCRIPTION
**Pull request by s1dds**
- Added new vulnerability 119-ios-harcoded-secrets.
- Added Japanese translations.

### CODE REVIEW

File: vulnerabilities/119-ios-harcoded-secrets/intro.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+This vulnerability arises because of the insecure practice to hardcode secrets, that includes API keys, server authentication credentials, in the application package.
```
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/intro.md:2]: There is no newline character at the end.

File: vulnerabilities/119-ios-harcoded-secrets/description.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+Malicious actors could take advantage of the hardcoded secrets to access private databases, leading to data breaches and the exposure of users' personal data.
```
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/description.md:2]: There is no newline character at the end.

File: vulnerabilities/119-ios-harcoded-secrets/business_implication.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+Hardcoded secrets can allow a malicious actor to access private resources that may contain sensitive information. This could lead to data breaches and the exposure of users' personal data which may cause financial losses to the organization.
```
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/business_implication.md:2]: There is no empty line at the end

**Comment by rajan262** [comment]
related_to.md file is empty. It should be filled with CWE information.

File: vulnerabilities/119-ios-harcoded-secrets/compliant.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/compliant.md:2]: This file should be empty

File: vulnerabilities/119-ios-harcoded-secrets/non_compliant.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/non_compliant.md:2]: This file should be empty

File: vulnerabilities/119-ios-harcoded-secrets/related_to.md
Comment (rajan262) [inline vulnerabilities/119-ios-harcoded-secrets/related_to.md:1]: Why is this file empty?

File: locales/ja/119-ios-harcoded-secrets/related_to.md
Comment (rajan262) [inline locales/ja/119-ios-harcoded-secrets/related_to.md:1]: This file should not be empty.

**Review by rajan262 — APPROVED** [review]
Looks good

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**